File: src/client/web/private/src/app/models/language-model.ts
import { IdCodeName } from '../value-objects/id-code-name';

export class LanguageModel {
    id: string;
    displayOrder: number;
    isApproved: boolean;
    version: number;
    creationTime: Date;
    lastModificationTime: Date;
    code: string;
    name: string;
    description: string;

}

File: src/server/Adfnet.Service/Models/LanguageModel.cs
﻿using System;
using Adfnet.Core;

namespace Adfnet.Service.Models
{
    public class LanguageModel : IServiceModel
    {
        public Guid Id { get; set; }
        public int DisplayOrder { get; set; }
        public bool IsApproved { get; set; }
        public int Version { get; set; }
        public DateTime CreationTime { get; set; }
        public DateTime LastModificationTime { get; set; }
        public string Code { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }

    }
}
